components/OrchestratorList/index.tsx

```diff
@@ -99,69 +141,457 @@ const OrchestratorList = ({ data, pageSize = 10 }) => {
+                    css={{
+                      borderBottom: "1px solid $neutral6",
+                      p: "$3",
+                    }}
+                  >
+                    <Text
+                      variant="neutral"
+                      size="1"
+                      css={{
+                        mb: "$2",
+                        fontWeight: 600,
+                        textTransform: "uppercase",
+                      }}
+                    >
+                      Assuming a delegation of:
+                    </Text>
+                    <Flex align="center">
+                      <TextField
+                        name="principle"
+                        placeholder="Amount in LPT"
+                        type="number"
+                        size="2"
+                        value={principle}
+                        onChange={(e) => {
+                          setPrinciple(
+                            Number(e.target.value) > maxSupplyTokens
+                              ? maxSupplyTokens
+                              : Number(e.target.value)
+                          );
+                        }}
+                        min="1"
+                        max={`${Number(
+                          protocolData?.totalSupply || 1e7
+                        ).toFixed(0)}`}
+                      />
+                    </Flex>
+                  </Box>
+                </PopoverContent>
+              </Popover>
+            </Flex>
+          </Tooltip>
+        ),
+        accessor: (row) => {
+          const pools = row.pools ?? [];
+          const rewardCallRatio =
+            pools.length > 0
+              ? pools.filter((r) => r?.rewardTokens).length / pools.length
+              : 0;
+
+          const activation = moment.unix(row.activationTimestamp);
+
+          const isNewlyActive = moment().diff(activation, "days") < 30;
+
+          const roi = calculateAnnualROI({
+            thirtyDayVolumeETH: Number(row.thirtyDayVolumeETH),
+            feeShare: Number(row.feeShare),
+            lptPriceEth: Number(protocolData.lptPriceEth),
+
+            yearlyRewardsToStakeRatio: Number(
+              protocolData.yearlyRewardsToStakeRatio
+            ),
+            rewardCallRatio: rewardCallRatio,
+            rewardCut: Number(row.rewardCut),
+            principle: Number(principle || 100),
+            totalStake: Number(row.totalStake),
+          });
+
+          return { roi, activation, isNewlyActive };
+        },
+        id: "projectedEarningsAPY",
+        Cell: ({ row }) => {
+          const isNewlyActive = useMemo(
+            () =>
+              moment().diff(
+                row.values.projectedEarningsAPY.activation,
+                "days"
+              ) < 30,
+            [row.values.projectedEarningsAPY.activation]
+          );
+
+          return (
+            <Popover>
+              <PopoverTrigger disabled={isNewlyActive} asChild>
+                <Badge
```

Comment:
Let's add a chevron icon to the badge so it's more obvious a user can click. <img width="154" alt="image" src="https://user-images.githubusercontent.com/555740/170293421-e6142a01-8843-4577-9b45-b2ea5aaacc05.png">